lang/rust/Makefile

```diff
@@ -0,0 +1,7 @@
+target :=helloworld.rs
```

Comment:
I would remove the `target` variable.

Simply add:
```
all: helloworld

helloworld: helloworld.rs:
        rustc -C link-args=-fPIC --target x86_64-unknown-linux-musl $<
```

The `build` target will be gone as well.